### DISCUSSION TITLE
OA-powered chatbot for finding content in documentation

### DISCUSSION THREAD

**Discussion by dustinwloring1988** [LAION-AI/Open-Assistant#3397] [Ideas]
Here is a little help documentation ai bot that was made for another project: https://www.youtube.com/watch?v=Yhtjd7yGGGA 
It would be nice to have something like this for this project but that uses OpenAssistant instead of ChatGPT. 

Let me know your guys thoughts

**Comment by olliestanley** [comment]
I am not sure we really have enough documentation currently to make this worthwhile, but we could consider it in future

**Comment by andreaskoepf** [comment]
If this issue is not only about OA documentation but product documentation and manuals in general then it could be related to retrieval: #3058

**Comment by sfkeller** [comment]
I am very impressed by your work. It's crazy that as a Swiss evangelist of "Open Whatever" I only now came across the projects LAION and Open-Assistant. 

Speaking of Semantic Search and Question Answering (over private documents) I assume you know "How to implement Q&A against your documentation with GPT3, embeddings and Datasette": https://simonwillison.net/2023/Jan/13/semantic-search-answers/ .
